server/src/routes/index.ts

```diff
@@ -35,6 +35,17 @@ router.use(cookieParser());
 router.use('/api-docs', swaggerUI.serve);
 router.get('/api-docs', swaggerUI.setup(openapiSpecification));
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function handleErrors(fn: any): any {
```

Comment:
Added TODO comment and moved to middlewares. Return cannot be void without breaking the whole thing 🙃

I was thinking about the function name. Is `handleErrors` good name since its more like passing on the possible errors and the `errorHandler` is the one actually processing errors. atm its really easy to mix these to naming wise. Would name like `routeDispatcher`, `routeHandler` or smth else be better?

Reply:
That's true, I don't think `route` in the name is ideal though. If I were to decide on a name right now I would pick  `controllerDispatcher`, but this is something that could be discussed more later. Not for this PR though since the sprint is going to end before we'd have a good chance to do so, so just place some reminder to come back to this.

Reply:
`controllerDispatcher` sounds good, updated to that.